### ISSUE TITLE
Crash on open repository of a submodule

### ISSUE THREAD

**Issue by CharlesB2** [rowanj/gitx#84]
With last version if I open a repo being a submodule (.git is file pointing to superproject's gitdir), GitX crashes with following message:

```
Assertion failed: (ref_out && repo && name), function git_reference_lookup_resolved, file /Users/build/jenkins/workspace/GitX-dev/objective-git/libgit2/src/refs.c, line 1082.
```

Complete crash report: https://gist.github.com/3706832

**Comment by rowanj** [comment]
This may be fixed by the changes by merged with issue #86, in GitX-dev-51+ (now building, should be available via in-app update imminently)

**Comment by CharlesB2** [comment]
Sorry for the dupe, fixed, thanks!
